status-im/status-react
# Fix WalletConnect header alignment on Android

fixes #20759
fixes #20859

### Summary

Fixed alignment issues with the wc header, which are mostly noticeable on Android. Also, added max width (via container-style) to quo/summary-tag, so it never goes out of the screen.

### Testing notes
Make sure summary/tag usage is not broken, cause I added some small changes there. 

### Steps to test


- Open Status on Android
- Send any WC request
- Observe the header



### Screenshots

![Screenshot 2024-07-23 at 15 24 00](https://github.com/user-attachments/assets/2360df92-6b68-4265-9a6a-2bb142d67c72)
![Screenshot 2024-07-23 at 15 23 31](https://github.com/user-attachments/assets/1c07d731-bb00-470a-a8ee-6476032474f9)

status: ready
